__tests__/data/salesforce_router_metadata_input.json

```diff
@@ -0,0 +1,98 @@
+{
+  "batchedRequest": {
+    "version": "1",
+    "type": "REST",
+    "method": "POST",
+    "endpoint": "https://ap15.salesforce.com/services/data/v50.0/sobjects/Lead",
+    "headers": {
+      "Content-Type": "application/json",
+      "Authorization": "Bearer 00D2v000002lXbX!ARcAQJBSGNA1Rq.MbUdtmlREscrN_nO3ckBz6kc4jRQGxqAzNkhT1XZIF0yPqyCQSnezWO3osMw1ewpjToO7q41E9.LvedWY"
```

Comment:
We shouldn't use the actual token, make sure you're not using the actual token

Reply:
Yeah, it was an expired token. We haven't any access to this.

Reply:
I'd still recommend to not use actual tokens in tests

Reply:
Yeah Sure, I'll update it.

Reply:
done